provisioning_client/devdoc/azure_provisioning_faq.md

```diff
@@ -0,0 +1,39 @@
+# Azure Iot Provisioning Client FAQ
+
+Here is where we shall try to answer question pertaining to the IoT Provisioning Client.  Please read the following documents for an overview of the DPS service.
+
+## x509 Provisioning
+
+- How does Provisining match the x509 certificates to an enrollment?
+
+  - The Provisioning Client add the x509 certificate (more on the certificate in a second) on the TLS stream.  The DPS service uses this certificate to look up the proper enrollment record.
```

Comment:
There are two parts here: how the provisioning client uses the certificate, and how DPS actually matches the device to the right enrollment. Please see my email to Randy for a walkthrough + diagram for how we match to an enrollment (which the doc team also has and I thought would have turned into a doc)

Reply:
also you have "more on this in a moment" twice in the FAQ so far (second time is below) - is there any way to be a little more specific as to when it's coming up?